downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,144 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Introduction of the alternate receive action
+
+The alternate receive action can be used to receive values corresponding to multiple send actions. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the result.
+
+```ballerina
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    worker w1 {
+        2 -> function;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> function;
+    }
+
+    worker w3 returns error? {
+        int value = 10;
+        if value == 10 {
+            return error("Error in worker 3");
+        }
+        value -> function;
+    }
+
+    worker w4 {
+        runtime:sleep(2);
+        3 -> function;
+    }
+
+    // The value of the variable `result` is set as soon as the value from either
```

Comment:
Move closer to usage?

Reply:
Can we? same as https://github.com/ballerina-platform/ballerina-dev-website/pull/8902#discussion_r1567199378. Named workers have to be declared first right?

Reply:
Ah, yes.